WORKSPACE

```diff
@@ -0,0 +1,94 @@
+workspace(name = "mujoco")
```

Comment:
This is using the [soon-to-be deprecated](https://bazel.build/external/migration) WORKSPACE system. It might be worth migrating this to use the new Bzlmod setup. I'll take a look at this if I have some extra time in the upcoming days. No guarantees though.

Reply:
Good point. I noticed that some dependencies are already in [Bazel Central Registry](https://registry.bazel.build/). Let me know how it goes - I’m happy to explore this as well.